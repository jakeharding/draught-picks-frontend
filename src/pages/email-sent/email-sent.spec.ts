/**
 * email-sent.spec.ts
 *
 * Created by jake
 * Created on 10/27/18
 *
 * The page seen after a confirmation email has been sent.
 */

describe('EmailSentPage', () => {

});