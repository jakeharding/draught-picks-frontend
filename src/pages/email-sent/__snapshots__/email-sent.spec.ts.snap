// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailSentPage should initialize the component 1`] = `
<page-email-sent
  navCtrl={[Function Object]}
  resend={[Function Function]}
>
  
  
  <draught-header />
  
  <ion-content>
    
    
    <ion-row>
      
      
      <ion-col
        class="below-header"
        col-12=""
        col-lg-6=""
        col-md-8=""
        col-sm-10=""
        col-xl-4=""
        offset-lg-3=""
        offset-md-2=""
        offset-sm-1=""
        offset-xl-4=""
        text-center=""
      >
        
        
        <h1>
          Email Sent
        </h1>
        
        
        <div
          class="justify"
        >
          
          
          <p
            margin=""
          >
            
            An email has been sent to the address you provided. Please use the link in the email to confirm your ownership of the email address.
          
          </p>
          
          
          <p
            margin=""
          >
            
            The link in the email is good for 24 hours.
          
          </p>
          
  
          
          <p
            margin=""
          >
            
            If you have trouble confirming the email click 
            <a>
              this link to resend it
            </a>
            .
          
          </p>
          
        
        </div>
        
      
      </ion-col>
      
    
    </ion-row>
    
  
  
  </ion-content>
  
</page-email-sent>
`;
